process_testcase.go

```diff
@@ -32,7 +36,7 @@ func (v *Venom) parseTestCase(ts *TestSuite, tc *TestCase) ([]string, []string,
 
 		var step TestStep
 		if err := yaml.Unmarshal([]byte(content), &step); err != nil {
-			return nil, nil, err
+			return nil, nil, errors.Wrapf(err, "unable to unmarshal teststep with content: %v", content)
```

Comment:
this error is not very readable, i would prefer to not repeat the content

Reply:
yes, it was for debugging, I remove it